src/patternfly/layouts/Flex/flex.scss

```diff
@@ -33,8 +33,30 @@ $pf-l-flex--variable-map: build-variable-map("pf-l-flex--spacer", $pf-l-flex--sp
   // reset spacer
   --pf-l-flex--spacer: var(--pf-l-flex--spacer-base);
 
+  order: var(--pf-l-flex--item--Order);
```

Comment:
did you mean to leave `--pf-l-flex--item--Order` unassigned? It doesn't appear to break anything, but we typically assign a value to a base var, even if it's just the default/initial value. Like `--pf-l-flex--item--Order: 0;` or `--pf-l-flex--item--Order: initial;`